spark_matcher/deduplicator/deduplicator.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, spark_session: SparkSession, col_names: Optional[List[str]] =
             raise ValueError(f"Invalid cluster_linkage_method: {cluster_linkage_method}")
         self.cluster_linkage_method = cluster_linkage_method
         # set the checkpoints directory for graphframes
-        self.spark_session.sparkContext.setCheckpointDir('checkpoints/')
+        self.spark_session.sparkContext.setCheckpointDir(checkpoint_dir)
```

Comment:
these are different checkpointers. i understand the confusion. graphframes (which is used under the hood) requires a checkpoint directory. That is the one we set hard coded. In Spark-Matcher, we use a checkpointer that we create ourselves to break the lineage in the DAG. For that checkpointer we use the checkpoint_dir as the argument in the init. As not to confuse the user to set two checkpointers, I would propose to change the checkpoint dir for graphframes (`self.spark_session.sparkContext.setCheckpointDir('checkpoints/')`) to an absolute path like `self.spark_session.sparkContext.setCheckpointDir('/tmp/checkpoints/')`. Could you please check if that works on databricks?

Reply:
I see, that makes sense. I will implement this change and create a new PR on a dedicated branch, as I realised that I opened this one on main.